src/nouislider.ts

```diff
@@ -168,6 +169,19 @@ export class NouisliderComponent implements ControlValueAccessor, OnInit {
     });
   }
 
+  ngOnChanges(changes: any) {
+    if (this.slider && (changes.min || changes.max)) {
+      setTimeout(() => {
+        this.slider.updateOptions({
+          range: {
+            min: this.min,
+            max: this.max
```

Comment:
I was initially doing that but rapid clicks on the change buttons caused exceptions inside angular.

This was where I found the fix http://stackoverflow.com/questions/38930183/angular2-expression-has-changed-after-it-was-checked-binding-to-div-width-wi